### DISCUSSION TITLE
Public appearance view color?

### DISCUSSION THREAD

**Discussion by digital-md** [community/community#88647] [New to GitHub]
I'm new here, my page appearance is set to black. But I noticed it says for my own view. But when I sign out and go view it on another browser it pulls up white. How do I make the appearance stay black?

**Comment by Shahnawazk7944** [comment]
hey @digital-md
The dark mode you set in your GitHub settings only applies to how you view your or other repositories on GitHub, not your own pages. When someone views your profile, their own settings will apply there.

**Comment by AlessioBugetti** [comment]
> I'm new here, my page appearance is set to black. But I noticed it says for my own view. But when I sign out and go view it on another browser it pulls up white. How do I make the appearance stay black?

Hello @digital-md , the dark mode settings on GitHub that you’ve set are personalized for your account. So when you’re logged in, you see everything in dark mode because that’s how you’ve set it up, but this is just for your view. When you sign out or if someone else visits your GitHub page and they’re not logged in, they’ll see the standard light theme. That’s the default for GitHub. If other people want to see the dark theme, they have to be logged into their own accounts and have set dark mode as their preference. There’s no way to set dark mode as the default view for your GitHub page for everyone, it depends on each user’s individual settings.

**Reply by Shahnawazk7944** [reply]
I guess I also said the same thing, right?
